packages/logging/logging.js

```diff
@@ -55,7 +55,7 @@ var META_COLOR = 'blue';
 // ops per second (on Node 8) by caching "process.platform".
 var platform = process.platform;
 var platformColor = function (color) {
-  if (platform === 'win32') {
+  if (platform === 'win32' && !color.endsWith('Bright')) {
```

Comment:
This is might be a good idea since the `logging` package could be used by outside packages and apps though I'd originally avoided it because it cut the ops per second in half over the original implementation.  It's possible I was being overzealous with optimizations, since even with this change `platformColor` still achieves 15MM ops/sec, but this is a logging function and I didn't want to introduce too much additional friction.

Reply:
15M ops/sec is still huge!